### ISSUE TITLE
Compatibility issues when trying to install

### ISSUE THREAD

**Issue by BenjaminBorn** [TuringLang/AdvancedMH.jl#22]
Hello,

I tried to install `AdvancedMH.jl` but ran into the following error:
```Julia
(v1.3) pkg> add AdvancedMH
 Resolving package versions...
ERROR: Unsatisfiable requirements detected for package Requires [ae029012]:
 Requires [ae029012] log:
 ├─possible versions are: [0.5.0-0.5.2, 1.0.0-1.0.1] or uninstalled
 ├─restricted by compatibility requirements with Plots [91a5bcdd] to versions: [0.5.0-0.5.2, 1.0.0-1.0.1]
 │ └─Plots [91a5bcdd] log:
 │   ├─possible versions are: [0.12.1-0.12.4, 0.13.0-0.13.1, 0.14.0-0.14.2, 0.15.0-0.15.1, 0.16.0, 0.17.0-0.17.4, 0.18.0, 0.19.0-0.19.3, 0.20.0-0.20.6, 0.21.0, 0.22.0-0.22.5, 0.23.0-0.23.2, 0.24.0, 0.25.0-0.25.3, 0.26.0-0.26.3, 0.27.0-0.27.1, 0.28.0-0.28.4, 0.29.0-0.29.7] or uninstalled
 │   └─restricted to versions 0.29.7 by an explicit requirement, leaving only versions 0.29.7
 ├─restricted by compatibility requirements with AdvancedMH [5b7e9947] to versions: 1.0.0-1.0.1
 │ └─AdvancedMH [5b7e9947] log:
 │   ├─possible versions are: [0.1.0, 0.2.0, 0.3.0-0.3.1, 0.4.0-0.4.1] or uninstalled
 │   ├─restricted to versions * by an explicit requirement, leaving only versions [0.1.0, 0.2.0, 0.3.0-0.3.1, 0.4.0-0.4.1]
 │   └─restricted by compatibility requirements with Distributions [31c24e10] to versions: [0.3.0-0.3.1, 0.4.0-0.4.1] or uninstalled, leaving only versions: [0.3.0-0.3.1, 0.4.0-0.4.1]
 │     └─Distributions [31c24e10] log:
 │       ├─possible versions are: [0.16.0-0.16.4, 0.17.0, 0.18.0, 0.19.1-0.19.2, 0.20.0, 0.21.0-0.21.3, 0.21.5-0.21.12, 0.22.0-0.22.5] or uninstalled
 │       └─restricted to versions 0.22.5 by an explicit requirement, leaving only versions 0.22.5
 └─restricted by compatibility requirements with Flatten [4c728ea3] to versions: 0.5.0-0.5.2 — no versions left
   └─Flatten [4c728ea3] log:
     ├─possible versions are: [0.0.1-0.0.4, 0.1.0, 0.2.0-0.2.1, 0.3.0-0.3.2] or uninstalled
     └─restricted to versions 0.1.0 by an explicit requirement, leaving only versions 0.1.0
```
Is this something you can fix on your side?

My system:
```
Julia Version 1.3.1
Commit 2d5741174c (2019-12-30 21:36 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin18.6.0)
  CPU: Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, skylake)
Environment:
  JULIA_EDITOR = atom  -a
  JULIA_NUM_THREADS = 4
```

**Comment by cpfiffer** [comment]
The issue here appears to be `Flatten`, since it does not support `Requires` 1.0. We could fix this by allowing Requires 0.5.2, but I'd rather not low the bounds to a pre-1.0 version. I'd open a pull request there and ask them to upgrade their `Requires` dependency to include 1.0.

However, it [doesn't appear](https://github.com/rafaqz/Flatten.jl/blob/master/Project.toml) that Flatten needs `Requires`, so I'm not sure what's up here. Have you tried updating everything before adding `AdvancedMH`?

**Comment by BenjaminBorn** [comment]
Thanks for your quick reply. I tried updating before adding `AdvanceMH` but apparently `Flatten` was stuck at an old release. I now managed to update it by hand and now `AdvancedMH` installs without an issue. Thanks again and sorry for the noise.
